src/installers.ts

```diff
@@ -643,19 +682,103 @@ const byPriority = (a: Installer, b: Installer) => a.priority - b.priority;
 //
 // Using Vortex parameter names here for convenience.
 //
-export const installerPipeline: Installer[] = [
+export const installerPipeline: InstallerWithPriority[] = [
+  {
+    type: InstallerType.CET,
+    id: "cp2077-cet-mod",
+    testSupported: testForCetMod,
+    install: installCetMod,
+  },
+  /*
+  {
+    type: InstallerType.Redscript,
+    id: "cp2077-redscript-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.Red4Ext,
+    id: "cp2077-red4ext-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.TweakDB,
+    id: "cp2077-tweakdb-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.AXL,
+    id: "cp2077-axl-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.INI,
+    id: "cp2077-ini-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.Config,
+    id: "cp2077-config-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.Reshade,
+    id: "cp2077-reshade-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.LUT,
+    id: "cp2077-lut-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.CoreCET,
+    id: "cp2077-core-cet-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.CoreRedscript,
+    id: "cp2077-core-redscript-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.CoreRed4ext,
+    id: "cp2077-core-red4ext-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  {
+    type: InstallerType.CoreCSVMerge,
+    id: "cp2077-core-csvmerge-mod",
+    testSupported: notSupportedModType,
+    install: notInstallableMod,
+  },
+  */
   {
     type: InstallerType.ArchiveOnly,
     id: "cp2077-basic-archive-mod",
-    priority: 30,
-    testSupported: modWithArchiveOnly,
-    install: archiveOnlyInstaller,
+    testSupported: testForArchiveOnlyMod,
+    install: installArchiveOnlyMod,
   },
   {
     type: InstallerType.Other,
     id: "cp2077-standard-mod",
-    priority: 31,
-    testSupported: modHasBadStructure,
-    install: installWithCorrectedStructure,
+    testSupported: testAnyOtherModFallback,
+    install: installAnyModWithBasicFixes,
+  },
+  {
+    type: InstallerType.NotSupported,
+    id: "cp2077-standard-mod",
```

Comment:
Pretty sure the id needs to be unique, even if the whole purpose of this step is to fail.

Reply:
Yep, it should!